apollo-router/src/plugins/rhai.rs

```diff
@@ -90,17 +92,14 @@ use rhai::plugin::*; // a "prelude" import for macros
 
 #[export_module]
 mod rhai_plugin_mod {
-    use super::RhaiContext;
-
-    pub(crate) fn get_operation_name(context: &mut RhaiContext) -> String {
-        todo!()
-        /*
-        match &context.context.request.body().operation_name {
-            Some(n) => n.clone(),
-            None => "".to_string(),
-        }
-            */
-    }
+    // use super::RhaiContext;
+
+    //     pub(crate) fn get_operation_name(context: &mut RhaiContext) -> String {
+    //         match &context.originating_request.body().operation_name {
+    //             Some(n) => n.clone(),
+    //             None => "".to_string(),
+    //         }
+    //     }
```

Comment:
This is something to look at after we merge this stuff, so leaving it commented out is the right thing for now.